addons/fleet/models/fleet_vehicle.py

```diff
@@ -295,7 +295,7 @@ def act_show_log_cost(self):
 
     def _track_subtype(self, init_values):
         self.ensure_one()
-        if 'driver_id' in init_values:
+        if 'driver_id' in init_values or 'future_driver_id' in init_values:
```

Comment:
i don't think you want this change in this pr

Reply:
oups ... sorry